packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2670,3 +2680,68 @@ def get(self, id):
         )
         tracks = get_user_tracks_remixed(query_args)
         return success_response(list(map(extend_track, tracks)))
+
+
+USER_PURCHASED_CONTENT_ROUTE = USER_ROUTE + "/purchased"
+
+purchased_response = make_response(
+    "purchased", ns, fields.List(purchasable_content_model)
+)
+
+full_purchased_response = make_full_response(
+    "full_purchased", full_ns, fields.List(purchasable_content_full_model)
+)
+
+
+@ns.route(USER_PURCHASED_CONTENT_ROUTE)
+class UserTracksPurchased(Resource):
+    @ns.doc(
+        id="""Get User Purchased Content""",
+        description="Gets content owned by the user which has been purchased by another user",
+        params={"id": "A User ID"},
+        responses={200: "Success", 400: "Bad request", 500: "Server error"},
+    )
+    @ns.expect(pagination_with_current_user_parser)
+    @ns.marshal_with(purchased_response)
+    def get(self, id):
+        decoded_id = decode_with_abort(id, ns)
+        args = pagination_with_current_user_parser.parse_args()
+        query_args = GetUserPurchasedContentArgs(
+            user_id=decoded_id,
+            current_user_id=get_current_user_id(args),
+            limit=get_default_max(args.get("limit"), 10, 100),
+            offset=get_default_max(args.get("offset"), 0),
+        )
+        purchased_content = get_user_purchasable_content(query_args)
+        purchased_content = list(map(extend_purchasable_content, purchased_content))
+
+        return success_response(
+            list(map(make_polymorph_purchasable_content, purchased_content))
+        )
+
+
+@full_ns.route(USER_PURCHASED_CONTENT_ROUTE)
+class FullUserTracksPurchased(Resource):
+    @full_ns.doc(
+        id="""Get User Purchased Content""",
```

Comment:
Just going to throw out that this will convert to `getUserPurchasedContent` and that sounds ambiguous to me given what it's actually returning (user-owned content that has been purchased). Not sure there's a better name and it has a description. Just flagging that it's a little cryptic.